### PR TITLE
[com_actionlogs] - fix article link when edited from frontend

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #22551 .

### Summary of Changes
translate to backend url

### Testing Instructions
#22551

### CODE REVIEW

File: administrator/components/com_actionlogs/helpers/actionlogs.php [hunk 1]
```diff
@@ -179,6 +179,12 @@ public static function getHumanReadableLogMessage($log)
 	 */
 	public static function getContentTypeLink($component, $contentType, $id, $urlVar = 'id')
 	{
+		if ($contentType === "form")
+		{
+			// Translate to backend
+			$contentType = "article";
```
Comment (Quy) [inline administrator/components/com_actionlogs/helpers/actionlogs.php]: Please use single quote.

File: administrator/components/com_actionlogs/helpers/actionlogs.php [hunk 2]
```diff
@@ -179,6 +179,12 @@ public static function getHumanReadableLogMessage($log)
 	 */
 	public static function getContentTypeLink($component, $contentType, $id, $urlVar = 'id')
 	{
+		if ($contentType === "form")
```
Comment (Quy) [inline administrator/components/com_actionlogs/helpers/actionlogs.php]: Single quote.

File: administrator/components/com_actionlogs/helpers/actionlogs.php (hunk 1 above)
Comment (alikon) [reply administrator/components/com_actionlogs/helpers/actionlogs.php]: done

File: administrator/components/com_actionlogs/helpers/actionlogs.php (hunk 2 above)
Comment (alikon) [reply administrator/components/com_actionlogs/helpers/actionlogs.php]: done

**Comment by alikon** [comment]
3 lines of code 2 @Quy yellow card collected :face_with_head_bandage: 
 :rofl:

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 25258a35dda0cd9f12cddb1a57dd0c6bf01b14ed<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22554">issues.joomla.org/tracker/joomla-cms/22554</a>.</sub>

**Comment by SharkyKZ** [comment]
Perhaps move this to the plugin? And add component check?

**Comment by infograf768** [comment]
@alikon 
@SharkyKZ is right. This should depend on the plugin.

I propose this as a solution.

```
diff --git a/plugins/actionlog/joomla/joomla.php b/plugins/actionlog/joomla/joomla.php
index 0fb7e27..2d06e50 100644
--- a/plugins/actionlog/joomla/joomla.php
+++ b/plugins/actionlog/joomla/joomla.php
@@ -78,5 +78,12 @@
 		}
 
-		list(, $contentType) = explode('.', $params->type_alias);
+		if ($option === 'com_content')
+		{
+			$contentType = 'article';
+		}
+		else
+		{
+			list(, $contentType) = explode('.', $params->type_alias);
+		}
 
 		if ($isNew)
@@ -191,5 +198,12 @@
 		}
 
-		list(, $contentType) = explode('.', $params->type_alias);
+		if ($option === 'com_content')
+		{
+			$contentType = 'article';
+		}
+		else
+		{
+			list(, $contentType) = explode('.', $params->type_alias);
+		}
 
 		switch ($value)
@@ -470,4 +484,9 @@
 		$option = $this->app->input->getCmd('option');
 
+		if ($table->get('module') != null)
+		{
+			$option = 'com_modules';
+		}
+
 		if (!$this->checkLoggable($option))
 		{

```

**Comment by infograf768** [comment]
Oh, I see it contains the check for com_modules which is already taken care in another PR.
I will merge that one now.

**Comment by infograf768** [comment]
@alikon
can you modify PR?
RC is for tomorrow

**Pull request closed without merging**